src/testUtils/globalMocks.ts

```diff
@@ -14,14 +14,16 @@ export const globalMocks = {
     Float: () => faker.random.number({ precision: 0.01 }),
     Boolean: () => faker.random.boolean(),
     ID: () => nextId(),
+    ISO8601DateTime: () => faker.date.past(10, '2020-01-14').toUTCString(),
+    ISO8601Date: () => faker.date.past(10, '2020-01-14').toUTCString(),
```

Comment:
I think we want this one just to be the date. I was trying to see if we could do it without moment but the answers seemed more complicated.
```suggestion
    ISO8601Date: () => moment(faker.date.past(10, '2020-01-14')).format('YYYY-MM-DD'),
```